README.md

```diff
@@ -6,8 +6,12 @@ of [DFT-FE](https://github.com/dftfeDevelopers/dftfe).
 
 To use these scripts, we assume you have cloned this
 repository onto a system where you intend to install DFT-FE.
-I installed it into `/lustre/orion/stf006/world-shared/$USER/DFT-FE`,
```

Comment:
Installing into world-shared allows other users to run the software without re-compiling everything here.